app/main.py

```diff
@@ -1,3 +1,73 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    @property
+    def km(self) -> int | float:
+        return self._km
+
+    @km.setter
+    def km(self, value: int | float) -> None:
+        if not isinstance(value, (int | float)):
```

Comment:
well done with this! just need a small fix to the second argument to isinstance method. (how do we separate items in tuple?)

Reply:
True, totally missed it, already fixed, will push with the next commit.